extensions/amp-pinterest/0.1/amp-pinterest.css

```diff
@@ -51,28 +51,52 @@
+  background-position-x: 4px;
+  background-size: 15px 15px;
+  border-radius: 4px;
+  max-height: 28px;
+  max-width: 56px;
+  min-height: 28px;
+  min-width: 56px;
+  text-indent: 23px;
+  text-decoration: none;
+  font: 12px/28px "Helvetica Neue", Helvetica, Arial, "sans-serif";
+  font-weight: bold;
+  -webkit-font-smoothing: antialiased;
 }
 
-.-amp-pinterest-ja-gray {
-  background-image: url(https://s-passets.pinimg.com/images/pidgets/pinit_bg_ja_rect_gray_20_2.png);
+.-amp-pinterest-en-gray, .-amp-pinterest-en-gray-tall, .-amp-pinterest-ja-gray, .-amp-pinterest-ja-gray-tall {
+  color: #fff;
+  background-color: #888;
 }
 
-.-amp-pinterest-ja-red {
-  background-image: url(https://s-passets.pinimg.com/images/pidgets/pinit_bg_ja_rect_red_20_2.png);
+.-amp-pinterest-en-red, .-amp-pinterest-en-red-tall, .-amp-pinterest-ja-red, .-amp-pinterest-ja-red-tall {
+  color: #fff;
+  background-color: #e60023;
 }
 
-.-amp-pinterest-ja-white {
-  background-image: url(https://s-passets.pinimg.com/images/pidgets/pinit_bg_ja_rect_white_20_2.png);
+.-amp-pinterest-en-white, .-amp-pinterest-en-white-tall, .-amp-pinterest-ja-white, .-amp-pinterest-ja-white-tall {
+  color: #e60023;
+  background-color: #fff;
+  box-sizing: border-box;
+  box-shadow: inset 0 0 1px #e60023;
```

Comment:
Is the inset shadow spec'd somewhere by pinterest?

Reply:
Not explicitly spec'd but followed the style from inspecting on the Widget Builder: https://developers.pinterest.com/tools/widget-builder/?type=follow